### ISSUE TITLE
sometimes ungit makes it seem like commits are on a single subtree when they are not

### ISSUE THREAD

**Issue by capaj** [FredrikNoren/ungit#1093]
I have this tree:

![image](https://user-images.githubusercontent.com/1305378/40223320-e8361758-5a82-11e8-8a90-a69fdb5ccf53.png)

which ungit renders such as it seems like the dev-1032 is continuing from to-asyn-await branch, which it is not.
See this rendering of the same tree when I check out dev-1032: 
![image](https://user-images.githubusercontent.com/1305378/40223396-3a8e77ca-5a83-11e8-9a31-bc8a71321d5d.png)

It would be nice if ungit could render the tree branches more separated. Maybe coloring the connections between commits as well as the commits would help?

**Comment by jung-kim** [comment]
Yes this is an issue and it is bit hard to envision a branch spacing that can remove this issue perfectly and consistently for all cases.  

I'ver personally tried different spacings and rendering logics but none seemed like a clear winner.

Therefore, I'm going to have it so that when a mouse hover over the line would highlight the line and the two nodes at the end of the line so one can easily demystify any confusion in case of doubt with a mouse hover.

How does that sound?

**Comment by capaj** [comment]
That would certainly be a step in the right direction. Even though it's unfortunate that in order to discriminate those branches, I'll have to hover over it, but certainly better than the current state.
